### PR TITLE
Bringing Facbebook\PersistentData namespace into PSR2 compliance

### PR DESCRIPTION
**Pull request by stevenmaguire**
It's happening.

### CODE REVIEW

**Comment by SammyK** [comment]
:+1: It's happening!

File: src/Facebook/PersistentData/FacebookMemoryPersistentDataHandler.php [hunk 1]
```diff
@@ -29,28 +29,26 @@
...
-    $this->sessionData[$key] = $value;
-  }
+    /**
+     * @inheritdoc
+     */
+    public function get($key)
+    {
+        return isset($this->sessionData[$key])
+            ? $this->sessionData[$key]
+            : null;
```
Comment (yguedidi) [inline src/Facebook/PersistentData/FacebookMemoryPersistentDataHandler.php]: One line IMO

Comment (stevenmaguire) [inline src/Facebook/PersistentData/FacebookMemoryPersistentDataHandler.php]: It's a close call there. That line is 82 characters long on a single line. The linter does not object, so I refactored to one line.

screenshot

Comment (yguedidi) [inline src/Facebook/PersistentData/FacebookMemoryPersistentDataHandler.php]: FYI, there is no hard limit in PSR-2 ;)

Comment (stevenmaguire) [inline src/Facebook/PersistentData/FacebookMemoryPersistentDataHandler.php]: I know. I just like to keep lines short and legible.

Comment (yguedidi) [inline src/Facebook/PersistentData/FacebookMemoryPersistentDataHandler.php]: For me, a ternary is more readable in one line, and it should always be in one line.
If it's too long, it mean you shouldn't use a ternary, but a classic `if`/`else`.
It's my personal tought

Comment (SammyK) [inline src/Facebook/PersistentData/FacebookMemoryPersistentDataHandler.php]: I like wrapping my ternary's. :)

**Pull request merged**